cirq/protocols/json_serialization.py

```diff
@@ -159,7 +159,7 @@ def two_qubit_matrix_gate(matrix):
                 'SycamoreGate': cirq.google.SycamoreGate,
                 'TaggedOperation': cirq.TaggedOperation,
                 'ThreeDQubit': cirq.pasqal.ThreeDQubit,
-                'TrialResult': cirq.TrialResult,
+                'Result': cirq.Result,
```

Comment:
for backwards compatibility, both "Result" and "TrialResult" should be present and point to `cirq.Result`